cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/RequiredLoginCommandOptions.java

```diff
@@ -0,0 +1,74 @@
+package cloud.foundry.cli.services;
+
+import static picocli.CommandLine.Option;
+
+import picocli.CommandLine;
+
+/**
+ * Options for the initialization process of the
+ * {@link org.cloudfoundry.operations.CloudFoundryOperations operations} object.
+ *
+ * @see cloud.foundry.cli.crosscutting.mapping.CfOperationsCreator
+ */
```

Comment:
The documentation should mention that the user is forced to provide target information for these `LoginCommandOptions`